c/tskit/tables.h

```diff
@@ -764,6 +786,21 @@ next operation that modifies the table (e.g., by adding a new row), but not afte
 int tsk_individual_table_get_row(
     tsk_individual_table_t *self, tsk_id_t index, tsk_individual_t *row);
 
+/**
+@brief Set the metadata schema
+
+@rst
+Copies the metadata schema string to this table, replacing any existing.
+@endrst
+
+@param self A pointer to a tsk_individual_table_t object.
+@param metadata_schema A pointer to a char array
+@param metadata_schema_length A pointer to a char array
```

Comment:
Typo here, repeated the previous description.

Reply:
Fixed https://github.com/tskit-dev/tskit/pull/493/commits/363b49d124ec69470922162f204d2d12ff0d0b9e